Library/Homebrew/locale.rb

```diff
@@ -68,6 +68,11 @@ def eql?(other)
   end
   alias == eql?
 
+  def detect(locale_groups)
```

Comment:
Last thing would be a test in `locale_spec.rb` for the `zh-TW`/`zh` example mentioned in https://github.com/Homebrew/homebrew-cask/issues/42755.

Reply:
Done.